app/components/TuneCard/index.js

```diff
@@ -49,16 +50,22 @@ export function TuneCard({ maxwidth, artistName, collectionName, cardImg, previe
       audioRef.current.pause();
     }
     setPlay(!play);
-    handleOnActionClick(audioRef);
+    if (handleOnActionClick) {
+      handleOnActionClick(audioRef);
+    }
   };
 
   return (
     <ItemCard maxwidth={maxwidth} data-testid="tune-card">
-      <StyledT fontSize={16} data-testid="artist-name">
+      <StyledT fontSize={1.6} data-testid="artist-name">
         {artistName}
       </StyledT>
-      <Image src={cardImg} width="100%" preview={false} />
-      <StyledT fontSize={18}>{collectionName}</StyledT>
+      <If condition={songId} otherwise={<Image src={cardImg} width="100%" preview={false} />}>
+        <Link to={`/songs/${songId}`}>
```

Comment:
Let's say I wanted to click on all parts of the card expect the button to route to the detail page. Can you make that change?